documentation/app/templates/components/list-item-template.hbs

```diff
@@ -0,0 +1,856 @@
+              <Label.Title>Title</Label.Title>
+              <Label.Metadata>
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+              </Label.Metadata>
+            </T.Label2>
+          </Cut::ListItem::Template>
+          <p>Small<br />(Mobile)</p>
+          <Cut::ListItem::Template
+            class="doc-list-item-template-types_small"
+            as |T|
+          >
+            <T.Label as |Label|>
+              <Label.Icon @name="hexagon" />
+              <Label.Title>Title</Label.Title>
+              <Label.Metadata>
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+              </Label.Metadata>
+            </T.Label>
+            <T.Badge @icon="hexagon" @text="Default" />
+            <T.Label2 as |Label|>
+              <Label.Icon @name="hexagon" />
+              <Label.Title>Title</Label.Title>
+              <Label.Metadata>
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+                <Cut::TextWithIcon @icon="hexagon" @text="IconLabel" />
+              </Label.Metadata>
+            </T.Label2>
+          </Cut::ListItem::Template>
+        </div>
+        <h2 id="component-anatomy">Component anatomy</h2>
+        <p>The List Item Template is made up from nested components. Below is
+          the anatomy for the individual components. For more details on their
+          respective APIs, view the component API section under the code tab.</p>
+
+        <h3 id="section">Label</h3>
+        <p>The List Item Template Label is the primary part of the template. It
+          consists of a title, as well as an option icon for the title and a
+          section to place metadata like badges, labels, etc. There are two
+          available Labels in the List Item Template. The first is required and
+          is available via the
+          <CodeInline @code="Label" />
+          contextual component. The optional second Label can be used via the
+          <CodeInline @code="Label2" />
+          contextual component.</p>
+        <img
+          src="/images/list-item-template-section.svg"
+          alt="List Item Template Section Anatomy"
+        />
+
+        <h3 id="connection-method">Connection Method</h3>
+        <p>The List Item Template also provides two different components for the
+          Connection Method between the two sections,
+          <CodeInline @code="Icon" />
+          and
+          <CodeInline @code="Badge" />. Both using the
+          <Hds::Link::Inline
+            @href="https://helios.hashicorp.design/icons/usage-guidelines"
+          >Icon</Hds::Link::Inline>
+          and
+          <Hds::Link::Inline
+            @href="https://helios.hashicorp.design/components/badge"
+          >Badge</Hds::Link::Inline>
+          components from HDS respectively.</p>
```

Comment:
Update copy to: 
"The List Item Template provides two different component options for the Connection Method, an `Icon` and a`Badge`. Both using the [Icon](https://helios.hashicorp.design/icons/usage-guidelines) and [Badge](https://helios.hashicorp.design/components/badge) components from HDS respectively. The connection Method option is only available if the second section is visible and will be displayed between each section."